### ISSUE TITLE
Using `with_advisory_lock!`

### ISSUE THREAD

**Issue by kbaum** [ClosureTree/with_advisory_lock#72]
Looks like a new method `with_advisory_lock!` was added which raises an error if a lock failed to be acquired but it's not yet available on the latest version. Is the latest code stable enough to cut a release for that?

Thanks.

**Comment by VladislavShved** [comment]
the same

**Comment by enclaver** [comment]
Yeah, this and `disable_query_cache`

**Comment by krschacht** [comment]
I ran into this too: the README documentation on the github repo is ahead of the published gem.

**Comment by dzar-aker** [comment]
Yep, you need to load it directly from here if you want to use the latest version:

in Gemfile:

`gem 'with_advisory_lock', git: "https://github.com/ClosureTree/with_advisory_lock.git" # "~> 4.6.0"`

then 

`bundle install`

**Comment by ssuttner** [comment]
@mceachen @seuros +1 on getting a new version out there, if it would be possible of course...
Thanks in advance!

**Comment by jeremyw** [comment]
Is this library even maintained any more?

**Comment by seuros** [comment]
version 5.0.0 was released.
